File: VTVApp.Api/Models/DTOs/Inspections/InspectionDetailsDto.cs
﻿namespace VTVApp.Api.Models.DTOs.Inspections
{
    public class InspectionDetailsDto
    {
        public Guid Id { get; set; }
        public Guid VehicleId { get; set; }
        public DateTime InspectionDate { get; set; }
        public List<InspectionCheckpointDetailDto> Checkpoints { get; set; }
        public string OverallComments { get; set; } // Any general comments from the inspector
        public int Status { get; set; } // Detailed status information
    }
}

File: VTVApp.Web/src/types/dtos/Inspections/InspectionDetailsDto.ts
import { InspectionCheckpointDetailDto } from "./InspectionCheckpointDetailDto"

export interface InspectionDetailsDto {
    id: string,
    vehicleId: string, 
    inspectorName: string,
    inspectionDate: string,
    checkpoints: InspectionCheckpointDetailDto[]
    overallComments: string,
    status: number
}